admin_frontend/src/components/ApiKeys.jsx

```diff
@@ -80,16 +83,16 @@ const ApiKeysPage = () => {
 				SUPPORTED_NETWORKS: supportedNetworks ?? "",
 				ERC20_PAYMASTERS: customErc20Paymaster ?? "",
 				TRANSACTION_LIMIT: txnMode,
-        NO_OF_TRANSACTIONS_IN_A_MONTH: noOfTxn,
-        INDEXER_ENDPOINT: 'http://localhost:3003'
+				NO_OF_TRANSACTIONS_IN_A_MONTH: noOfTxn,
+				INDEXER_ENDPOINT:
+					process.env.REACT_APP_INDEXER_ENDPOINT ?? "http://localhost:3003",
```

Comment:
Should `http://localhost:3003` not also be a environment configuration?

Reply:
thats what I wrote it as process.env.REACT_APP_INDEXER_ENDPOINT